### PR TITLE
Script to delete objects versions for a list of runs

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/CMakeLists.txt [hunk 1]
```diff
@@ -424,7 +424,8 @@ install(FILES example-default.json
         DESTINATION etc)
 
 install(PROGRAMS script/RepoCleaner/o2-qc-repo-cleaner
-                 script/RepoCleaner/o2-qc-repo-deletion
+                 script/RepoCleaner/o2-qc-repo-delete-runs
+                 script/RepoCleaner/o2-qc-repo-delete-period
```
Comment (knopers8) [inline Framework/CMakeLists.txt:428]: I was thinking that you mean data taking period (as the metadata `PeriodName`). Could you please clarify this in the executable name? E.g.  `o2-qc-repo-delete-interval`, o2-qc-repo-delete-time-period`?

File: Framework/script/RepoCleaner/o2-qc-repo-delete-runs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    logging.debug("")
+                elif answer.lower() in ["n", "no"]:
+                    logging.info("   skipping")
+                else:
+                    logging.error("   wrong input, skipping")
+            else:
+                ccdb.deleteVersion(v)
+                logging.debug("")
+
+        logging.info(f"Deleted items: {len(versions)}")
```
Comment (knopers8) [inline Framework/script/RepoCleaner/o2-qc-repo-delete-runs:91]: the count will also include the versions where you said "no". That could be misleading and scare us in the future.

**Review by knopers8 — APPROVED** [review]
thanks, it's good like this, but please see my suggestions for now or for the next release.

**Comment by Barthelemy** [comment]
I agree on both comments and will modify the script tomorrow

**Pull request merged**